src/components/dropdown/dropdown.tsx

```diff
@@ -199,55 +188,36 @@ export const Dropdown: FC<DropdownProps> = ({
     ));
 
   return (
-    <Manager>
+    <div ref={containerRef} className={cx('container', className)} title={title}>
+      <button
+        {...getToggleButtonProps({
+          tabIndex: disabled ? -1 : 0,
+          className: cx('dropdown', variant, toggleButtonClassName, {
+            'transparent-background': transparentBackground,
+            opened: isOpened,
```

Comment:
it's not critical, but we could rename this hook as well to 
```suggestion
            const [opened, setOpened] = useState(false);
```